impeller/display_list/aiks_dl_blend_unittests.cc

```diff
@@ -246,6 +246,62 @@ TEST_P(AiksTest, ImageFilterBlend) {
   ASSERT_TRUE(OpenPlaygroundHere(callback));
 }
 
+TEST_P(AiksTest, ColorFilterAdvancedBlend) {
```

Comment:
Nit, maybe a link to the bug you filled for maximum context.

Reply:
done